pkg/services/sqlstore/org_users.go

```diff
@@ -14,6 +14,7 @@ func init() {
 	bus.AddHandler("sql", AddOrgUser)
 	bus.AddHandler("sql", RemoveOrgUser)
 	bus.AddHandler("sql", GetOrgUsers)
+	bus.AddHandler("sql", SearchOrgUsers)
```

Comment:
We are moving away from the bus, and prefer instead calling methods on relevant services (this avoids global state etc), in this case SQLStore, directly. Could you refactor to call `hs.SQLStore.SearchOrgUsers` from `HTTPService` instead of adding a bus handler?